src/pymovements/datasets/dataset.py

```diff
@@ -598,9 +635,16 @@ def save_events(self, events_dirname: str | None = None, verbose: int = 1):
                 print('Save file to', events_filepath)
 
             events_filepath.parent.mkdir(parents=True, exist_ok=True)
-            event_df_out.write_ipc(events_filepath)
-
-    def save_preprocessed(self, preprocessed_dirname: str | None = None, verbose: int = 1):
+            if extension == 'feather':
+                event_df_out.write_ipc(events_filepath)
+            elif extension == 'csv':
+                event_df_out.write_csv(events_filepath)
```

Comment:
add an else clause and raise a `ValueError` in the `else` clause. use a text like this:
```
f'unsupported file format "{extension}". Supported formats are: {valid_extensions}'
```
with
```
valid_extensions = ['csv', 'feather']
```

Reply:
done